man/systemd.network.xml

```diff
@@ -992,6 +992,24 @@
             <para>Specifies the outgoing device to match. The outgoing interface is only available for packets originating from local sockets that are bound to a device.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SourcePort=</varname></term>
+          <listitem>
+            <para>Specifies the source port range match in fib rules. Default to unset.</para>
```

Comment:
Please explain the syntax of port range. How about the following?
"Specifies the source port or port range separated by match in forwarding information base (FIB) rules. A port range is specified by the lower and upper port separated by a dash."